x-pack/plugins/security_solution/public/flyout/entity_details/host_right/content.tsx

```diff
@@ -71,6 +78,8 @@ export const HostPanelContent = ({
         observedFields={observedFields}
         queryId={HOST_PANEL_OBSERVED_HOST_QUERY_ID}
       />
+      <EuiHorizontalRule />
```

Comment:
we probably don't want to show it either. Plus I'd move the logic of checking our indexes inside the new `InsightEntity` component. And would return `null` if there are no data. It later will have more insight blocks, this container component doesn't need to know about the internal logic of InsightEntity